CHANGELOG.md

```diff
@@ -12,6 +12,13 @@ This release will bump the Realm file format from version 22 to 23. Opening a fi
 * [Sync] `App.close()` have been added so it is possible to close underlying ressources used by the app instance.
 * [Sync] Add support for progress listeners with `SyncSession.progress`. (Issue [#428](https://github.com/realm/realm-kotlin/issues/428))
 * Add better error messages when inheriting `RealmObject` with unsupported class types. (Issue [#1086](https://github.com/realm/realm-kotlin/issues/1086))
+* Added support for `@PersistedName` annotations for mapping a Kotlin field name to the underlying field name persisted in the Realm. (Issue [#590](https://github.com/realm/realm-kotlin/issues/590))
+  ```kt
```

Comment:
we don't have code snippets in the Changelog but leave that for the source code documentation. I would remove this